### PR TITLE
Add search for a pattern in a file

### PR DESCRIPTION
**Pull request by adlajebara**
Assignment 3

### CODE REVIEW

**Comment by MissZarah** [comment]
Easy and good solution. 
Leave comments for how Pattern works.

File: search.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+public class search {
+	
+	
+	public static void main(String[] args) throws IOException {
+		
+	
+		BufferedReader read = new BufferedReader(new FileReader(args[2])); 
+   
+        Pattern pattern = Pattern.compile(args[1]);
```
Comment (MissZarah) [inline search.java:16]: It is good to create a method for this part

File: search.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	
+	public static void main(String[] args) throws IOException {
+		
+	
+		BufferedReader read = new BufferedReader(new FileReader(args[2])); 
+   
+        Pattern pattern = Pattern.compile(args[1]); 
+  
+        String line; 
+        while ((line = read.readLine()) != null) {
```
Comment (MissZarah) [inline search.java:19]: create a seperate method for this one

File: search.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class search {
+	
+	
+	public static void main(String[] args) throws IOException {
+		
+	
+		BufferedReader read = new BufferedReader(new FileReader(args[2]));
```
Comment (MissZarah) [inline search.java:14]: read should be args[1] and pattern args[0]

**Review by MissZarah — CHANGES_REQUESTED** [review]

File: search.java (hunk 3 above)
Comment (adlajebara) [reply search.java]: Changes have been made!

**Review by MissZarah — APPROVED** [review]

**Pull request merged**